pages/advocates/index.vue

```diff
@@ -79,10 +79,25 @@ main {
   justify-content: space-around;
 }
 
-@media (max-width: 720px) {
+.card {
```

Comment:
You added the `advocate-card` class to the component. Use that to refer to the component instead of `card` (don't forget removing it from the component HTML too).